Podfile.lock

```diff
@@ -52,6 +52,6 @@ SPEC CHECKSUMS:
   SwiftLint: 7a0227733d786395817373b2d0ca799fd0093ff3
   Turf: c6bdf62d6a70c647874f295dd1cf4eefc0c3e9e6
 
-PODFILE CHECKSUM: 7bcf4f19f8cf7317dcf4c7f8b9c4b19129f1e386
+PODFILE CHECKSUM: cc584b88f9c4c9951551a518e17e7bbc885c1e57
 
-COCOAPODS: 1.6.1
+COCOAPODS: 1.5.3
```

Comment:
Please update to CocoaPods 1.6.1 and redo these changes. Using `bundle` to invoke `pod` and `fastlane` commands (e.g., `bundle install && bundle exec pod install`) prevents these sorts of inadvertant downgrades.